discovery_engine_core/core/src/storage.rs

```diff
@@ -111,6 +117,13 @@ pub(crate) trait FeedbackScope {
         document: document::Id,
         reaction: UserReaction,
     ) -> Result<ApiDocumentView, Error>;
+
+    async fn update_time_spend(
```

Comment:
🔧 past time, as in the engine
```suggestion
    async fn update_time_spent(
```